docs-ref-conceptual/TOC.yml

```diff
@@ -200,6 +219,9 @@
         href: azure-cli-vm-tutorial-6.md
       - name: 7 - Summary
         href: azure-cli-vm-tutorial-7.md
+  - name: Use persisted parameters
+    href: param-persist-tutorial.md
```

Comment:
This file has been removed in https://github.com/MicrosoftDocs/azure-docs-cli/pull/3933